gui/default/index.html

```diff
@@ -551,8 +551,9 @@ <h4 class="panel-title">
                       <a href="#" class="toggler" ng-click="toggleUnits()">
                         <span ng-if="!metricRates">{{connectionsTotal.inbps | binary}}B/s</span>
                         <span ng-if="metricRates">{{connectionsTotal.inbps*8 | metric}}bps</span>
-                        ({{connectionsTotal.inBytesTotal | binary}}B)</span>
+                        ({{connectionsTotal.inBytesTotal | binary}}B)
                       </a>
+                      <small ng-if="config.options.maxRecvKbps > 0"><br/><i class="text-muted"><span translate>Limit</span>: {{config.options.maxRecvKbps}} KiB/s</i></small>
```

Comment:
I am pretty sure `maxRecvKbps` are actually bits, not bytes. And this should follow the rate limit unit, i.e. change unit on toggle as well.

Reply:
I entered "123" in the settings (it says KiB/s there) and "123" is displayed in the GUI line I added.